src/plugins/mailer/test/index.test.ts

```diff
@@ -1,48 +1,149 @@
-import { FastifyInstance } from 'fastify';
+import { faker } from '@faker-js/faker';
 
-import { SUCCESS_MESSAGES } from '@graasp/translations';
+import { FastifyInstance } from 'fastify';
 
 import build, { clearDatabase } from '../../../../test/app';
 import { resolveDependency } from '../../../di/utils';
+import { MailBuilder } from '../builder';
+import { MAIL } from '../langs/constants';
+import enTranslations from '../langs/en.json';
 import { MailerService } from '../service';
 
 describe('Mailer', () => {
   let app: FastifyInstance;
   let mailerService: MailerService;
+  let mockSendEmail;
 
   beforeAll(async () => {
     ({ app } = await build());
     mailerService = resolveDependency(MailerService);
   });
 
-  afterAll(async () => {
+  beforeEach(async () => {
+    mockSendEmail = jest
+      .spyOn(mailerService, 'sendEmail')
+      .mockImplementation(async () => Promise.resolve());
+  });
```

Comment:
Does this need to be re-created before each test ? Can't it be defined once in the `beforeAll` block and simply reset in the afterEach (already done)?

Reply:
Indeed.